cibuildwheel/windows.py

```diff
@@ -365,7 +445,14 @@ def build(options: Options, tmp_path: Path) -> None:
 
                 repaired_wheel = next(repaired_wheel_dir.glob("*.whl"))
 
-            if build_options.test_command and options.globals.test_selector(config.identifier):
+            if cross_compiling_target_arch:
+                log.step("skip wheel testing (cross compile)")
```

Comment:
The same thing that's done for macOS universal/arm should be done here; the printout about skipping should only happen if this test is selected (which it is by default).